include/stumpless/prival.h

```diff
@@ -0,0 +1,62 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/*
+ * Copyright 2018-2023 Joel E. Anderson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @file
+ * Function for converting prival from int to string
+ */
+
+#ifndef __STUMPLESS_PRIVAL_STRING
```

Comment:
the header guard should match the others in the project: this one should be `__STUMPLESS_PRIVAL_H`

Reply:
I've fixed the header guard to match the rest of the project.